prov/util/src/util_buf.c

```diff
@@ -39,6 +39,20 @@
 #include <ofi.h>
 #include <ofi_osd.h>
 
+#ifdef HAVE_FABRIC_PROFILE
+#include <ofi_profile.h>
+static inline void
+ofi_bufpool_add_mem(size_t size)
```

Comment:
The name can be changed to reflect that it is tracking the change instead of "adding the memory" itself.  for example, `ofi_bufpool_track_add_mem` .